pyevmasm/__main__.py

```diff
@@ -65,10 +67,24 @@ def main():
                 if buf_set <= hex_set:  # subset
                     buf = binascii.unhexlify(buf)
 
+        # replace all swarm hashes with \x00
+        if args.no_swarmhash:
+            while True: #detect multiple swarm hashes, about 700 contracts do
+               if '\xa1ebzzr0X ' in buf:
+               # replace the swarm hash with stops
+                   offset = buf.find('\xa1ebzzr0X ')
+                   buf = buf[:offset] + b'\x00' * 43 + buf[offset+43:]
+               else:
+                   break
+
         insns = list(disassemble_all(buf))
         for i in insns:
-            args.output.write("%08x: %s\n" % (i.pc, str(i)))
+            if args.no_offsets:
+                args.output.write("%s\n" % (str(i),))
```

Comment:
the `(,)` is redundant so it could be `% str(i)` but I understand it might be better to leave it s it is.
We could also use `"{}\n".format(i)`